src/components/layouts/collection-page-layout.tsx

```diff
@@ -682,7 +682,7 @@ const Fresh = ({freshness}: {freshness: any}) => {
           {freshness.title && (
             <h2 className="text-xl font-semibold">
               {freshness.status === 'fresh' && '🌱'}
-              {freshness.status === 'stale' && '⛔️'}
+              {freshness.status === 'stale' && '⚠️'}
```

Comment:
the little ⚠️ doesn't read nicely and you'd need to use an svg or something, which is why I used ⛔️